roles/bmc_fw_update/tasks/main.yml

```diff
@@ -113,10 +171,6 @@
     category: Update
     command: GetUpdateStatus
     baseuri: "{{ inventory_hostname }}"
-    username: "{{ dpu_bmc_username }}"
```

Comment:
@abhashsolanki18 why this is removed ?

Reply:
@abhashsolanki18 should this be either `username` and `password` or `auth_token` ?
OR
can we apply ` | default(omit)`  everywhere above instead of creating new block ? 

WDYT ?

Reply:
@glimchb 
applied default (omit) everywhere that should work and simplify the code by removing the blocks.
see the latest commit

Reply:
ok, make sure you test it works... I had issues with this before...

Reply:
it's working perfectly for me